imports/api/Constants.js

```diff
@@ -34,7 +34,14 @@ export const Constants = {
 		// Japanese Riichi Style
 		JAPANESE: "jpn"
 	},
-	MAHJONG_CLUB_LEAGUE: "Mahjong Club League"
+
+	EAST: "east",
+	SOUTH: "south",
+	WEST: "west",
+	NORTH: "north",
+
+	MAHJONG_CLUB_LEAGUE: "Mahjong Club League",
+	MANGAN: 2000
```

Comment:
You have good intentions here, but "MANGAN = 2000" is meaningless.  You may as well have called the constant "Y2K" or "Andre minus 1000".  At least call this something like MANGAN_BASE_POINTS or something.